### PR TITLE
Code Refactoring

### PR DESCRIPTION
**Pull request by Gauravbhatt19**
@vijay-singh07 Please have a review of this PR

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
+        <div id="main">
+
+            <?php if ( isset( $error_msg ) ) : ?>
+                <h1><?php echo $error_msg; ?></h1>
+            <?php endif; ?>
+
+            <h1>SIMPLE LOGIN</h1>
+            <form method="POST">
+                <label>Username:<input type="text" name="username" class="text" autocomplete="off" required></label>
+                <label>Password:<input type="Password" name="password" class="text" required></label>
```
Comment (Gauravbhatt19) [inline index.php]:
```suggestion
                <label>Password:<input type="password" name="password" class="text" required></label>
```

File: index.php [hunk 2]
```diff
@@ -1,44 +1,48 @@
...
+                <input type="Submit" name="Submit" id="Sub">
```
Comment (Gauravbhatt19) [inline index.php]:
```suggestion
                <input type="submit" name="submit" id="sub">
```

File: index.php (hunk 1 above)
Comment (vijay-singh07) [reply index.php:43]: a label of username and the other one for password,right?
Screenshot 2020-11-18 195241

File: index.php (hunk 2 above)
Comment (vijay-singh07) [reply index.php]: okay sir,noted!

File: connection.php [hunk 3]
```diff
@@ -1,7 +1,5 @@
 <?php
-   define('DB_SERVER', 'localhost');
-   define('DB_USERNAME', 'root');
-   define('DB_PASSWORD', '');
-   define('DB_DATABASE', 'login_form');
-   $db = mysqli_connect(DB_SERVER,DB_USERNAME,DB_PASSWORD,DB_DATABASE);
-?>
\ No newline at end of file
+
+	require_once 'project-config.php';
```
Comment (vijay-singh07) [inline connection.php:3]: image

**Pull request merged**